pom.xml

```diff
@@ -151,6 +151,26 @@
                         </headerDefinitions>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>xml-maven-plugin</artifactId>
+                    <version>${xml-maven-plugin.version}</version>
+                    <configuration>
+                        <indentSize>2</indentSize>
+                        <formatFileSets>
+                            <formatFileSet>
+                                <directory>${project.basedir}</directory>
+                                <includes>
+                                    <include>**/*.xml</include>
+                                </includes>
+                                <excludes>
+                                    <exclude>**/.*</exclude>
+                                </excludes>
+                            </formatFileSet>
```

Comment:
- [ ] You should exclude `target` directories.
Because these are not source directories.

When I execute `mvn xml:check-format` without project cleaning, error is occurred due to remaining `target` directory codes. This causes confusion.